r/tests/testthat/test-extra-package-roundtrip.R

```diff
@@ -65,6 +65,9 @@ test_that("readr read csvs roundtrip", {
 test_that("data.table objects roundtrip", {
   load_or_skip("data.table")
 
+  # https://rdatatable.gitlab.io/data.table/articles/datatable-importing.html#testing-using-testthat
+  .datatable.aware=TRUE
```

Comment:
This works, but note the linked advice is for this to show up in R/, not here in tests/

The advice you're following is here:

https://github.com/Rdatatable/data.table/blob/83fd2c05ce2d8555ceb8ba417833956b1b574f7e/R/cedta.R#L25-L27

Reply:
Thanks! I've updated this comment